src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -147,23 +149,30 @@ class LocalBackend extends Backend with SharedFileSystem with LazyLogging {
 
   private def runSubprocess(backendCall: BackendCall): ExecutionResult = {
     val tag = makeTag(backendCall)
-    val (_, stdoutWriter) = backendCall.stdout.fileAndWriter
-    val (_, stderrWriter) = backendCall.stderr.fileAndWriter
+    val stdoutTailed = backendCall.stdout.tailed(100)
+    val stderrTailed = backendCall.stderr.tailed(100)
```

Comment:
I'm having a hard time understanding why the `.tailed(100)` is here... where is it being used?